<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffLogin</class>
 <widget class="QDialog" name="StaffLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>444</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>登录系统</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 15px;
}


QPushButton {		
	font: 20px;
	color: rgb(0,0,0);
}

QLineEdit{
	background: rgb(232, 234, 221);
	border:0px;
	color:rgb(0, 0, 0);
	font: 20px;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_userName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>280</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>280</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>90</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>90</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>关  闭</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
